### ISSUE TITLE
Move docs from repo to yetibot.com

### ISSUE THREAD

**Issue by devth** [yetibot/yetibot#812]
https://github.com/yetibot/yetibot/blob/master/doc

**Comment by maplemuse** [comment]
What are your thoughts on just having the docs as part of cljdoc?

**Comment by devth** [comment]
Yes! I'm a big fan of the vision of cljdoc and I think it fills part of our docs needs. IIRC you were at the cljdoc talk where I asked @martinklepsch how he's thinking about the various styles of docs (API reference, tutorials or guides, blog posts, etc). The way I'm currently thinking about it is:

- API docs on cljdoc for sure
- Tutorial or guides - not sure. Could be markdown / cljdoc or could remain on yetibot.com
- Blog posts - stay on yetibot.com
- Yetibot.com continues to serve as the general marketing site where we try to quickly demonstrate its value

So next steps for me would be:

- [ ] get yetibot working on cljdoc (@martinklepsch gave me some tips on how to debug)
- [ ] look at current docstring coverage and expand, especially covering core/foundational aspects of Yetibot like parsing and evaluation pipeline
- [ ] ponder more about what stays on yetibot.com vs moving to cljdoc

Open to feedback on all of this!
